integreat_cms/cms/migrations/0016_rename_location_not_on_map_poi_location_on_map.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.12 on 2022-04-03 14:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Migration file to rename the "location on map" field of the POI model
+    """
+
+    dependencies = [
+        ("cms", "0015_user_unique_email_field"),
```

Comment:
```suggestion
        ("cms", "0020_alter_last_updated_field"),
```
In the meantime, there have been a few other migrations - and to ensure a linear migration history, we have to update the prior migration here. Also, could you increase the counter of this file and rename it to `0021_rename_location_not_on_map.py`? Thanks!